public_html/application/classes/Model/CADeclarerevolt.php

```diff
@@ -104,12 +104,10 @@ protected function check( $par, &$message )
 		// cooldown rivolta
 		
 		$res = ORM::factory('Battle')
-			->where( array( 
-			'type' => 'revolt',
-			'dest_region_id' => $par[1] -> region -> id,
-			'timestamp >' => time() - (KO7::$config->load('medeur.revolt_cooldown') * 24 * 3600 ) ) ) ;
+			->where('type', '=', 'revolt')
+			->and_where('dest_region_id', '=', $par[1] -> region -> id)
+			->and_where('timestamp >', '=', time() - (KO7::$config->load('medeur.revolt_cooldown') * 24 * 3600 ));
```

Comment:
for this `and_where` the operator is `>` you see the operator appending to the end of the first param string there?